lib/domain/app_state/search/server_search_state.dart

```diff
@@ -35,3 +35,10 @@ class ServerSearchChatFailed extends Failure {
   @override
   List<Object?> get props => [exception];
 }
+
+class ServerSearchLoading extends Success {
```

Comment:
IMO: I don't see you using it in `Interactor`, where to use it?

Reply:
okey, change in interactor from ServerSearchInitial => ServerSearchLoading